### PR TITLE
CTAT lncrna

### PR DESCRIPTION
**Pull request by asmariyaz23**
* [Yes ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [Yes ] This PR adds a new recipe.
* [Yes ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [No ] This PR updates an existing recipe.
* [No ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/ctat-lncrna/build.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# ctat_lncrna_DIR_NAME="$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM"
+# The above is an error. The following is correct.
+ctat_lncrna_DIR_NAME="$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM"
+ctat_lncrna_INSTALL_PATH="$PREFIX/share/$ctat_lncrna_DIR_NAME"
+# Make the install directory and move the ctat_lncrna files to that location.
+mkdir -p $PREFIX/bin
+mkdir -p $ctat_lncrna_INSTALL_PATH
+#copy to INSTALL_PATH
+cp -R $SRC_DIR/LICENSE.txt $SRC_DIR/lncrna $SRC_DIR/SciEDPipeR $ctat_lncrna_INSTALL_PATH
```
Comment (bgruening) [inline recipes/ctat-lncrna/build.sh]: why do you copy the license as well? Conda will do this for you if you have specified it in the meta.yaml, which you have done :)
Why not copying everything into PREFIX/bin directly?

File: recipes/ctat-lncrna/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{% set name = "ctat-lncrna" %}
+{% set version = "2.0" %}
+{% set build = "1" %}
```
Comment (bgruening) [inline recipes/ctat-lncrna/meta.yaml]: should be 0 if you have a new version

File: recipes/ctat-lncrna/build.sh (hunk 1 above)
Comment (asmariyaz23) [reply recipes/ctat-lncrna/build.sh]: The reason for not copying into bin directly is as follows:
Most of the CTAT tools use different versions of SciEDPipeR, so if a user installs 2 of CTAT tools then version conflicts may occur hence my attempt at keeping everything in install path and invoking from there.

**Comment by bgruening** [comment]
Ok, thanks!

File: recipes/ctat-lncrna/meta.yaml (hunk 2 above)
Comment (asmariyaz23) [reply recipes/ctat-lncrna/meta.yaml]: I am making a new build including the changes you suggested.

**Pull request merged**